### ISSUE TITLE
Send additional folksonomy tags

### ISSUE THREAD

**Issue by regorxxx** [phw/foo_listenbrainz2#24]
Now folksonomy tags may also be sent.
https://listenbrainz.readthedocs.io/en/latest/users/json.html#submission-json

Would it be possible to add them?

For ex. adding a new input field, where people could add semicolon-separated list of folksonomy tags to send:
![image](https://github.com/user-attachments/assets/ab33fbfc-82c9-44fa-beca-412ca78a17fc)

It would be pretty great since LB database could be easily filled with genres with this change.

**Comment by phw** [comment]
Sure, I just wonder how useful static tags are. My suggestion would have been to submit the genres as tags. Or maybe have an option to define a tag name to read the tags from the file (and it could default to "genre").

**Comment by regorxxx** [comment]
Yep, I was talking about the input field pointing to actual tags from file (not static values). Also agree with adding GENRE as default.

**Comment by phw** [comment]
Just to not forget about this: I had started implementing this in https://github.com/phw/foo_listenbrainz2/commit/25c9574fff16ab7d3cbe8374e46329d4fc4ffcd5 . This is just submitting the genres as folksonomy tags. Extending this with a UI option and a list of tags to collect would be the next step.

**Comment by redactedscribe** [comment]
> Sure, I just wonder how useful static tags are. My suggestion would have been to submit the genres as tags. Or maybe have an option to define a tag name to read the tags from the file (and it could default to "genre").

I use STYLE to contain the majority of my tags (GENRE contains the broader ones), so being able to configure which one to read would be nice indeed. Although, I'd likely not use this feature (track genres / styles are identical across a release for me, and they're pulled from MB to begin with, so submitting them back wouldn't serve any purpose; LB already displays some genre info from MB).

**Comment by regorxxx** [comment]
Hi, is there any update on this?
